academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md

```diff
@@ -19,13 +19,25 @@ You can find this information by logging into the Azure Management Portal, searc
 
 ## Azure Subscription quota limit reached
 
-If you encounter errors such as "operation could not be completed as it results in exceeding approved XXXXXX quota", you'll need to contact Azure support to increase your account quota, try a different region where you have more Azure quota, or change the MAKES host machine sku/worker machine sku settings. 
+If you encounter errors such as "operation could not be completed as it results in exceeding approved XXXXXX quota", you can to contact Azure support to increase your account quota, try a different region where you have more Azure quota, or change the MAKES host machine SKU/worker machine SKU parameter. 
 
-For most Microsoft Academic sub-graph index builds, you can scale down to **Standard_D8_v3** sku to build and host the indexes. MAKES indexing resources will scale using Azure Batch's "Dedicated Cores". Make sure you're requesting for the correct quota increase for the correct resource and region.
+When running **CreateHostResources** command:
+
+    Requires at least your subscription to have at least 4 avaliable vCPU cores and ability to create **Standard_DS4_v2** Virtual Machines. 
+
+When running **DeployHost** command:
+
+    By default, uses Virtual Machine SKU **Standard_D14_v2**. (Running the command by default requires your subscription to have access to at least 16 vCPU and ability to Virtual Machine with **Standard_D14_v2** SKU in west us region.) For most Microsoft Academic sub-graph index, you can scale down to **Standard_D8_v3** sku to host the indexes. 
```

Comment:
>For most Microsoft Academic sub-graph index, you can scale down to **Standard_D8_v3** sku to host the indexes. [](start = 236, length = 110)

"For most sub-graph indexes, you can scale down to use **Standard_D8_v3** SKU instances to host the indexes."